offices/_single.html

```diff
@@ -76,14 +89,18 @@ <h1 class="h3">{{ top_story.head | safe }}</h1>
     </section>
     {% endif %}
     {% if office.resources %}
-    <section class="office_resources block block__padded-top block__border-top">
+    <section class="office_resources
+                    block block__padded-top
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/e446711d95276e18e0a3d778a6040713215dd5c3